axonserver/src/main/java/io/axoniq/axonserver/component/version/Version.java

```diff
@@ -0,0 +1,72 @@
+package io.axoniq.axonserver.component.version;
+
+import static java.lang.Integer.parseInt;
+
+/**
+ * Represents an artifact version.
+ *
+ * @author Sara Pellegrini
+ * @since 4.2.1
+ */
+public interface Version {
+
+    String DOT_REGEX = "\\.";
```

Comment:
Shouldn't these have a little bit of javadoc too, as they're visible to users? Granted, it feel insignificant, but still.